pullzone_add_custom_certificate.go

```diff
@@ -0,0 +1,30 @@
+package bunny
+
+import (
+	"context"
+	"fmt"
+)
+
+type pullZoneAddCustomCertificateOptions struct {
+	Hostname       string `json:"Hostname"`
+	Certificate    []byte `json:"Certificate"`
+	CertificateKey []byte `json:"CertificateKey"`
```

Comment:
The json package encodes `[]byte` as base64 already, we don't have to manually encode anything. We load the content of the PEM-encoded key and certificate chain directly to `Certificate` and `CertificateKey`.

Bunny itself don't document the format they accept for certificates, so I discovered experimentally that x509 pem works for certificate and pem works for keys. Do you think I should document that? Maybe people should look at bunny documentation for that, since they may change the supported format. Maybe they support formats other then pem, I don't even know :)

Reply:
> The json package encodes []byte as base64 already, we don't have to manually encode anything. 

ah, great :+1: 

> Bunny itself don't document the format they accept for certificates, so I discovered experimentally that x509 pem works for certificate and pem works for keys. Do you think I should document that? 

I agree we should not document the supported cert formats here, to prevent that we have to keep it uptodate.